integration-tests/README.md

```diff
@@ -68,16 +68,48 @@ can either be 8 or 11.
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
 The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
 
+## Running tests by docker-compose + mvn
```

Comment:
We do not need to run build and docker-compose manually right?
If we run mvn verify -pl integration-tests -P integration-tests... then it should automatically build and run docker containers. If that is the case, I think this section can be rename to instruction for using/manually bringing up docker containers instead of running tests.

Reply:
on run integration tests you also run docker containers.
Renamed section